tests/lib/nsxt_unittest.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2023 The Capirca Project Authors All Rights Reserved.
+# Copyright 2023 VMware, Inc. SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""UnitTest class for nsxt.py."""
+
+import json
+import optparse
+from absl.testing import absltest
+
+from capirca.lib import naming
+from capirca.lib import nsxt
+from capirca.lib import policy
+from tests.lib import nsxt_mocktest
+
+
+class TermTest(absltest.TestCase):
+
+  def setUp(self):
+    """Call before every test case."""
+    super().setUp()
+    parser = optparse.OptionParser()
+    parser.add_option(
+        '-d',
+        '--def',
+        dest='definitions',
+        help='definitions directory',
+        default='def')
+    (FLAGS, args) = parser.parse_args([])
+    self.defs = naming.Naming(FLAGS.definitions)
+
+  def tearDown(self):
+    super().setUp()
+    pass
+
+  def runTest(self):
```

Comment:
Where does the need for `runTest` come from? absl tests should not require this.